### PR TITLE
1.7.0 prerelease branch merge to main

### PR DESCRIPTION
**Pull request by vhuseinova-msft**
# What

Prerelease branch merge to main

# Why

# How Tested

# Process & policy checklist

- [x] I have updated the project documentation to reflect my changes if necessary.
- [x] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/chat-component-bindings/CHANGELOG.json [hunk 1]
```diff
@@ -1,6 +1,33 @@
...
+    {
+      "date": "Mon, 14 Aug 2023 18:28:42 GMT",
+      "tag": "@internal/chat-component-bindings_v1.7.0",
+      "version": "1.7.0",
+      "comments": {
+        "patch": [
+          {
+            "author": "beachball",
+            "package": "@internal/chat-component-bindings",
+            "comment": "Bump @internal/acs-ui-common to v1.7.0",
```
Comment (emlynmac) [inline packages/chat-component-bindings/CHANGELOG.json:13]: This looks like it has duplicates?

File: change/@azure-communication-react-47707f0a-52e6-473f-89c0-fad8f3944926.json [hunk 2]
```diff
@@ -1,6 +1,8 @@
 {
   "type": "none",
-  "comment": "Fixed cursor position during mention editing",
+  "area": "improvement",
```
[the remark this answers is not in the document]
Comment (vhuseinova-msft) [reply change/@azure-communication-react-47707f0a-52e6-473f-89c0-fad8f3944926.json:3]: I think it's how the file is merged ( the name of the file is changed too)

File: packages/react-composites/CHANGELOG.json [hunk 3]
```diff
@@ -1,6 +1,57 @@
...
+      "date": "Mon, 14 Aug 2023 18:28:42 GMT",
+      "tag": "@internal/react-composites_v1.7.0",
+      "version": "1.7.0",
+      "comments": {
+        "patch": [
+          {
+            "author": "beachball",
+            "package": "@internal/react-composites",
+            "comment": "Bump @internal/acs-ui-common to v1.7.0",
+            "commit": "2e2916d59cca7d82e98f016276587353d3316597"
```
Comment (emlynmac) [inline packages/react-composites/CHANGELOG.json:14]: More duplicates here.

File: packages/storybook/CHANGELOG.json [hunk 4]
```diff
@@ -1,6 +1,27 @@
...
+            "author": "beachball",
+            "package": "@internal/storybook",
+            "comment": "Bump @azure/communication-react to v1.7.0",
+            "commit": "2e2916d59cca7d82e98f016276587353d3316597"
+          },
+          {
+            "author": "beachball",
+            "package": "@internal/storybook",
+            "comment": "Bump @internal/fake-backends to v1.7.0",
+            "commit": "2e2916d59cca7d82e98f016276587353d3316597"
```
Comment (emlynmac) [inline packages/storybook/CHANGELOG.json:20]: and here?

File: tools/check-treeshaking/CHANGELOG.json [hunk 5]
```diff
@@ -1,6 +1,51 @@
...
+            "author": "beachball",
+            "package": "@internal/check-treeshaking",
+            "comment": "Bump @internal/acs-ui-common to v1.7.0",
+            "commit": "2e2916d59cca7d82e98f016276587353d3316597"
+          },
+          {
+            "author": "beachball",
+            "package": "@internal/check-treeshaking",
+            "comment": "Bump @internal/react-components to v1.7.0",
+            "commit": "2e2916d59cca7d82e98f016276587353d3316597"
```
Comment (emlynmac) [inline tools/check-treeshaking/CHANGELOG.json:20]: Duplicate entries

File: packages/chat-component-bindings/CHANGELOG.json (hunk 1 above)
Comment (vhuseinova-msft) [reply packages/chat-component-bindings/CHANGELOG.json:13]: it has different comments for each item and they were generated automatically by beachball

Comment (emlynmac) [reply packages/chat-component-bindings/CHANGELOG.json:13]: OK, so we're expecting multiple refs to the same commit then?

**Review by emlynmac — APPROVED** [review]

File: packages/acs-ui-common/package.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@internal/acs-ui-common",
-  "version": "1.7.0-beta.1",
+  "version": "1.7.0",
```
Comment (JamesBurnside) [inline packages/acs-ui-common/package.json:3]: Package versions should be `1.7.1-beta.0`, without the beta prefix we don't get the option `prerelease` in the changelog command

File: .github/workflows/create-release-branch.yml [hunk 7]
```diff
@@ -93,8 +93,10 @@ jobs:
       - name: Synchronize package version reported to telemetry
         run: node common/scripts/sync-telemetry-package-version
 
-      - name: Add changelog
-        run: node ./common/scripts/changelog/change.mjs -m 'Create release branch and update SDK versions' --type none
+      - name: Disabled - DO THIS MANUALLY! Add changelog
```
Comment (JamesBurnside) [inline .github/workflows/create-release-branch.yml:96]: Are the docs updated to include doing this manually now?

Comment (vhuseinova-msft) [reply .github/workflows/create-release-branch.yml:96]: There is an ADO task to fix this and add this step back

Comment (vhuseinova-msft) [reply .github/workflows/create-release-branch.yml:96]: @JoshuaLai will be working on this, not sure if it will be done before the next release

Comment (JamesBurnside) [reply .github/workflows/create-release-branch.yml:96]: OK, make sure to notify the next release driver about this extra manual step

**Review by JamesBurnside — APPROVED** [review]

**Pull request merged**